### PR TITLE
Homepage re-design

### PR DESCRIPTION
**Pull request by shreyassanthu77**
# Progress

- [x] Header
- [ ] Features Section (WIP)
- [ ] Footer

> Sidenote: I also switched to TypeScript

### CODE REVIEW

**Comment by FilledStacks** [comment]
Awesome! I'll review soon. I wanted to switch to TypeScript but didn't want to break everything 😅 thanks.

**Comment by shreyassanthu77** [comment]
One more thing is this is still a very much work in progress and I am trying to make it a lot better

File: src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                  copied ? "opacity-100" : ""
+                }`}
+              >
+                Copied to Clipboard
+              </p>
+            </button>
+          </p>
+        </div>
+      </main>
+      <Features />
```
Comment (FilledStacks) [inline src/pages/index.tsx:107]: Please remove features form the home page

File: src/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        >
+          <GithubIcon className="mr-4" />
+          Github
+        </a>
+      </header>
+      <main className="px-6 py-1 min-h-[70vh] flex flex-col items-center justify-center">
+        <h1 className="text-5xl font-black text-center md:text-7xl">
+          The
+          <span className="text-transparent bg-clip-text bg-gradient-to-br from-indigo-800 to-purple-400">
+            scalable
```
Comment (FilledStacks) [inline src/pages/index.tsx:72]: Change the main title to "The [Flutter Framework] for Production" The text in `[]` should be the new purple. For production should be on the second line.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          {tagline}
+        </p>
+        <div className="flex flex-col items-center gap-4 px-6 py-4 md:flex-row">
+          <Link
+            to="/docs/getting-started/overview"
+            className="hover:text-white hover:no-underline px-6 py-4 text-lg font-semibold text-white transition-all bg-black border-none rounded outline-none cursor-pointer ring-transparent focus:ring-purple-500 ring hover:bg-[#444] active:scale-90"
+          >
+            Get Started
+          </Link>
+          <p className="flex items-center gap-4 p-4 py-2 m-0 ml-3 font-mono text-lg font-bold bg-gray-200 rounded">
```
Comment (FilledStacks) [inline src/pages/index.tsx:87]: I like this, but without context there it won't give the developers using it a good experience. For now it should be removed. New developers that get here won't have stacked installed so this won't work either.

File: src/pages/index.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      "dart pub global activate stacked_cli\nstacked create app awesome_app"
+    );
+    setCopied(true);
+
+    setTimeout(() => setCopied(false), 3000);
+  }, []);
+
+  return (
+    <>
+      <header className="sticky top-0 flex items-center justify-between px-6 py-4 shadow bg-white/20 backdrop-blur-md">
```
Comment (FilledStacks) [inline src/pages/index.tsx:54]: Could we remove the header from the landing page. I'd like getStarted to be the only option for the user, the rest of the docs can have the header with the Stacked icon and the github icon.

**Review by FilledStacks — APPROVED** [review]
Thank you so much for the contribution! I really appreciate it. 

There's a few changes I'd like before merging in, as mentioned below. 

I'm leaving it on the review so you can see, but since I'm busy with the docs I will merge it into a different branch, apply the changes and push it up.

**Pull request merged**